include/audit_events.hpp

```diff
@@ -38,7 +38,7 @@ inline void auditEvent([[maybe_unused]] const crow::Request& req,
         return;
     }
 
-    strcpy(cnfgBuff, opPath);
+    strncpy(cnfgBuff, opPath, std::strlen(opPath) + 1);
```

Comment:
I'm also wondering about the hard-coded 256 for cnfgBuff. Should this be a macro value tied to something well-known? Should strncpy_s() be used instead?